locksmith/src/models/verifier.ts

```diff
@@ -0,0 +1,16 @@
+import { Table, Column, Model } from 'sequelize-typescript'
+
+@Table({ tableName: 'Verifiers', timestamps: true })
+export class Verifier extends Model<Verifier> {
+  @Column
+  verifierAddress!: string
+
+  @Column
+  userAddress!: string
```

Comment:
I am confused here. what is the `userAddress`? Same as `verifierAddress` ?

Reply:
I think we are missing `lockAddress` ?

Reply:
added `lockAddress` and `verifierAddress` -> `address`

thk!